web/src/search/input/QueryInput.tsx

```diff
@@ -134,8 +134,9 @@ export class QueryInput extends React.Component<Props, State> {
                             .join(' ')
                         return fetchSuggestions(fullQuery).pipe(
                             map(createSuggestion),
+                            filter((suggestion): suggestion is Suggestion => !!suggestion),
```

Comment:
Has more accurate typings
```suggestion
                            filter(isDefined),
```